doc/src/site/sphinx/configuration/configuration_properties.rst

```diff
@@ -328,6 +328,10 @@ External backend configuration properties
 |                                                       |                | specifies memory for internal JVM   |
 |                                                       |                | use outside of Java heap.           |
 +-------------------------------------------------------+----------------+-------------------------------------+
+| ``spark.ext.h2o.external.communication.blockSize``    | ``1m``         | The size of blocks representing     |
```

Comment:
imho would be good to say what is the unit - that the block size is in bytes

Reply:
no problem, will add some comment. Thanks!

Reply:
Updated the description